### ISSUE TITLE
Location Search QHW6+CQ

### ISSUE THREAD

**Issue by vshcherb** [osmandapp/Osmand#7022]
QHW6+CQ Northern Administrative Okrug, Moscow

**Comment by mbirth** [comment]
Sounds like what was mentioned in #4958 - only full OLC / Plus codes work at the moment, not relative.

E.g. this should work:
```
9G7VQHW6+CQ
```

**Comment by vshcherb** [comment]
Yes but we are going to support relative to my location or to a city

**Comment by vshcherb** [comment]
So, QHW6+CQ Moscow - should work and QHW6+CQ (if you are in Moscow now) should work as well

**Comment by nikos2wheels** [comment]
The other day I was trying to navigate to a local restaurant by sharing the plus code from Google Maps, WX53+XG Washington, District of Columbia, and OsmAnd could not find the coordinate correctly. It was showing me a location that was 51 miles away. If I edited the plus code and removed the city and only entered the relative code WX53+XG it would find the address correctly. Also, if I entered the full code 87C4WX53+XG it would also find it correctly. I noticed the same behavior with other POI in the same area. I thought OsmAnd could handle plus codes shared from the Google Maps app. Why does it fail in this case?
